README.md

```diff
@@ -1,2 +1,64 @@
 # ibc-apps
-IBC applications and middleware for Cosmos SDK chains
+
+![IBC-APPS Header](ibc-apps.png)
```

Comment:
This is just a general comment which probably needs a bit more discussion - do we want their to be a feeling of coherence between ibc-apps and ibc-go? Right now the readme's have a different flow of content and different descriptions of ibc for example. If we want a unified feel, I would suggest we align on this and that isn't to say ibc-go just sets the tone but we can adapt both in parallel?

Reply:
I think we should aim for a unified feel in terms of readme, documentation etc. Could be good to have teams sync on this given that the [README revamp for ibc-go](https://github.com/cosmos/ibc-go/issues/1747) is still ongoing. cc @crodriguezvega